src/lib/common/JsonHelper.cpp

```diff
@@ -215,19 +220,38 @@ void JsonHelper::addNumber(const std::string& key, long long value)
   empty = false;
 }
 
+
+
+/* ****************************************************************************
+*
+* JsonHelper::addInt -
+*/
+void JsonHelper::addInt(const std::string& key, long long value)
```

Comment:
Just to confirm... the only actual changes in this files are addInt() and addBoolean() functions, all the rest is style and typo fixing. Is correct?

Reply:
Yes, that's it

Reply:
NTC